### PR TITLE
Add e2e basic tests and travis config

### PR DESCRIPTION
**Pull request by osdnk**
## Motivation
In this state test provided by me does not really check gesture-handler related features. However I see that there's need even for checking if example application is actually building.

## Features
I made some test with mocha and detox and they check some minor things (don't really test gesture handling). I've added building scripts on travis for iOS and Android. I also added running tests on travis on iOS. It was impossible to do it on Android because travis doesn't support HAXM (neither circleCI)

### CODE REVIEW

**Comment by kmagiera** [comment]
LGTM 👍

**Comment by kmagiera** [comment]
jk

**Comment by osdnk** [comment]
So here's the point when iOS story is finished. Let's start a new chapter of the adventure

**Comment by osdnk** [comment]
So here's android done. Now merge them

File: Example/android/build.gradle [hunk 1]
```diff
@@ -3,7 +3,7 @@
 buildscript {
     ext {
         buildToolsVersion = "27.0.3"
-        minSdkVersion = 16
```
Comment (osdnk) [inline Example/android/build.gradle]: It's just an example, but required by travis

File: Example/multitap/index.js [hunk 2]
```diff
@@ -11,38 +11,55 @@ import {
...
+              ref={this.doubleTapRef}
+              onHandlerStateChange={this._onDoubleTap}
+              numberOfTaps={2}>
+              <View style={styles.box} testID="rectangle" />
+            </TapGestureHandler>
           </TapGestureHandler>
-        </TapGestureHandler>
-      </LongPressGestureHandler>
+        </LongPressGestureHandler>
+        {this.state.lastGH && (
```
Comment (osdnk) [inline Example/multitap/index.js]: I didn't manage to catch modal text

**Comment by osdnk** [comment]
Revert example changes

File: Example/android/gradlew.bat [hunk 3]
```diff
@@ -1,90 +1,90 @@
-@if "%DEBUG%" == "" @echo off
```
Comment (osdnk) [inline Example/android/gradlew.bat]: It's a matter of linebreaking. It's not breaking

**Pull request merged**